### PR TITLE
fix: Stop creation of multiple listeners.

### LINKED ISSUE #1088: Multiple listeners getting created.
**Describe the bug**
Code for listener to loadevents is written in oncreateview. Thus, multiple listeners are getting created when we go from one fragment to another and then come back.

**To Reproduce**
1. Log when listeners are setup.(eg in eventdetailsfragment log in loadevents)
2. Click on something which takes to another fragement (eg tickets)
3. Press back.
4. You will see log appearing twice. Keep repeating you will get multiple logs.

**Expected behavior**
Listener should be setup only once when we come back

**Logs**
2019-02-13 15:30:39.278 28770-28770/com.eventyay.attendee E/message: loading - open
2019-02-13 15:30:43.229 28770-28770/com.eventyay.attendee E/message: loading - came back 1st time
2019-02-13 15:30:43.273 28770-28770/com.eventyay.attendee E/message: loading - came back 1st time
2019-02-13 15:30:43.278 28770-28770/com.eventyay.attendee E/message: loading - came back 1st time
2019-02-13 15:30:46.130 28770-28770/com.eventyay.attendee E/message: loading - came back 2nd time
2019-02-13 15:30:46.173 28770-28770/com.eventyay.attendee E/message: loading - came back 2nd time
2019-02-13 15:30:46.178 28770-28770/com.eventyay.attendee E/message: loading - came back 2nd time
2019-02-13 15:30:46.182 28770-28770/com.eventyay.attendee E/message: loading - came back 2nd time
2019-02-13 15:30:48.251 28770-28770/com.eventyay.attendee E/message: loading - came back 3rd time
2019-02-13 15:30:48.293 28770-28770/com.eventyay.attendee E/message: loading - came back 3rd time
2019-02-13 15:30:48.299 28770-28770/com.eventyay.attendee E/message: loading - came back 3rd time
2019-02-13 15:30:48.305 28770-28770/com.eventyay.attendee E/message: loading - came back 3rd time
2019-02-13 15:30:48.309 28770-28770/com.eventyay.attendee E/message: loading - came back 3rd time

**Would you like to work on the issue?**

- [x] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by angmas1**
Fixes #1088 

Changes: Moved code for creating listeners for which ever objects possible to onCreate instead of onCreateView so that they get created only once when fragment is created. Couldn't move some objects which relied on creation of rootview first.

### CODE REVIEW

**Comment by angmas1** [comment]
Yes, its a spotless kotlin error fixing it.

File: app/src/main/res/layout/fragment_events.xml [hunk 1]
```diff
@@ -99,6 +99,7 @@
                             android:id="@+id/eventsRecycler"
                             android:layout_width="match_parent"
                             android:layout_height="match_parent"
+                            android:paddingTop="@dimen/padding_medium"
```
Comment (iamareebjamal) [inline app/src/main/res/layout/fragment_events.xml]: Why?

Comment (angmas1) [reply app/src/main/res/layout/fragment_events.xml]: ShimmerEvents have have space near top which disappears when Events get loaded so added padding for consistency. Wanted to open another PR, but pushed in this by mistake. Do you want me to remove this or is it okay?
image

image

Comment (angmas1) [reply app/src/main/res/layout/fragment_events.xml]: Is this okay? @iamareebjamal

**Pull request merged**